src/main/app.ts

```diff
@@ -38,6 +39,7 @@ app.locals.ENV = env
 
 const developmentMode = env === 'development'
 
+app.use(noCache())
```

Comment:
Please use `app.use(helmet.noCache())` instead. Helmet also brings `nocache` dependency, no need to add it manually.

Reply:
Also best to add this to `Helmet.enableFor` in `modules/helmet/index.ts`.

Reply:
ok